include/lsst/jointcal/RefStar.h

```diff
@@ -15,31 +15,37 @@ namespace jointcal {
 //! Tangent Plane seems a good idea.
 class RefStar : public BaseStar {
 private:
-    unsigned int _index;
-    std::vector<double> _refFlux;
+    // on-sky flux, in Maggies, per filter
+    std::vector<double> _refFluxList;
+    std::vector<double> _refFluxErrList;
 
 public:
+    // RefStar(double xx, double yy, std::vector<double>& refFluxList, std::vector<double>& refFluxErrList)
+    //         : FatPoint(xx, yy), _refFluxList(refFluxList), _refFluxErrList(refFluxErrList) {}
+
+    RefStar(double xx, double yy, double defaultFlux, double defaultFluxErr, std::vector<double>& refFluxList,
+            std::vector<double>& refFluxErrList)
+            : BaseStar(xx, yy, defaultFlux, defaultFluxErr),
+              _refFluxList(refFluxList),
+              _refFluxErrList(refFluxErrList) {}
     //!
-    RefStar(const BaseStar& baseStar);
+    RefStar(const BaseStar& baseStar) : BaseStar(baseStar) {}
```

Comment:
In fact, I've explicitly deleted the all but the copy constructors for Fitted/MeasuredStars, and all from RefStar, since they all should be managed via shared_ptr in general.